### DISCUSSION TITLE
Matomo - Alternative to Google Analytics

### DISCUSSION THREAD

**Discussion by Nexulo** [area17/twill#1406] [Ideas]
For privacy reasons, I avoid all services from Google, Facebook etc. as much as possible. Therefore, an alternative to Google Analytics would be good. I use Matomo (free and open source -> [link](https://matomo.org/free-software/)), which we can install on our own server and so no data is passed to third parties.

Are there already plans for the implementation of Matomo? I'm sure many others would welcome this alternative. Matomo also has a good/simple API, so this should be possible.

**Comment by ifox** [comment]
Hi @maisen20, definitely understand your reasons, however want to make sure we are on the same page. The GA integration in Twill is disabled by default and is very light, it does not do anything else than display those graphs on the dashboard. Is that what you would like to see integrated with Matomo? If so, it should definitely we possible.

**Comment by Nexulo** [comment]
Hi @ifox,
exactly, I would like to have the statistics from Matomo in the dashboard. 
So the dashboard would also show me some statistics about the website visitors and i could also switch to my Matomo interface via a link. 

For Twill, I haven't programmed anything yet except for some small customizations for myself, but I think that should be relatively easy to incorporate with the Matomo API.
